controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -517,6 +551,14 @@ func (r *OCSInitializationReconciler) ensureOcsOperatorConfigExists(initialData
 		util.DisableCSIDriverKey:         strconv.FormatBool(true),
 	}
 
+	if util.EnableTopologyKey == "true" {
+		err = patchCsiDriver(r.ctx, r.Client, &r.Log, util.TopologyDomainLabelsKey, initialData.Namespace)
```

Comment:
util.TopologyDomainLabelsKey is again a string it doesn't contain the desired topology domain labels